### ISSUE TITLE
Update German translation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#367]
```
This patch adds German translations for new commands speaker/hangup/callspeaker 
and fixes translation of xmpp_manager_unreachable_servers.
```

Original issue reported on code.google.com by `quert.zuiopue` on 7 Sep 2014 at 12:45

Attachments:
- [0001-Translated-new-commands-speaker-hangup-callspeaker-t.patch](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-367/comment-0/0001-Translated-new-commands-speaker-hangup-callspeaker-t.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks
I've merged the translations.
They will be available in the next build.
```

Original comment by `Florent....@gmail.com` on 7 Sep 2014 at 4:13
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 20 Oct 2014 at 5:31
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
```
Released on Play Market as 5.0
```

Original comment by `Florent....@gmail.com` on 25 Oct 2014 at 6:23
- Changed state: **Deployed**
